applications/FluidDynamicsApplication/python_scripts/adjoint_fluid_analysis.py

```diff
@@ -3,7 +3,6 @@
 import KratosMultiphysics as Kratos
 import KratosMultiphysics.FluidDynamicsApplication as KFluid
 import KratosMultiphysics.HDF5Application as KHdf5
```

Comment:
@msandre Travis is failing bcs of this
=> you have to skip the tests using HDF5 in case it is not compiled

Reply:
thx for explaining :+1:  I will fix it.